### PR TITLE
#240 added USER vars for vater and rous repo, edited verbiage

### PR DESCRIPTION
**Pull request by Donovan-Medina**
#240 added USER vars for vater and rous repo, edited verbiage of copying ssh key to github, validated go downloads/installs correctly

### CODE REVIEW

**Review by marissaeinhorn — APPROVED** [review]
Looks good

**Review by rylagek — CHANGES_REQUESTED** [review]
line 162 needs to be updated to match your new verbiage - if rous is not already cloned, the setup script fails to clone it even with proper access permissions

File: control-services/bin/setup.sh [hunk 1]
```diff
@@ -114,7 +115,7 @@ echo
 ssh-add $SETUP_SSH_KEY_PATH
 
 echo
-echo "Copy this key to the $SETUP_REPO repo as a deploy key. Press any key when done."
+echo "Copy this key to github as an SSH key, call it $SETUP_REPO deploy. Press any key when done."
```
[the remark this answers is not in the document]
Comment (Donovan-Medina) [reply control-services/bin/setup.sh:118]: The key is not to a specific repo. It's part of github's settings of ssh keys -- they could theoretically could be called anything in github. 
image

File: control-services/bin/setup.sh [hunk 2]
```diff
@@ -158,7 +159,7 @@ if test -f /home/control/$CONFIG_REPO/.git/config; then
     echo
     echo "$CONFIG_REPO exists"
 else
-    git clone git@github.com:$ORG_OR_USER/$CONFIG_REPO.git /home/control/$CONFIG_REPO
+    git clone git@github.com:$CONFIG_REPO/$CONFIG_REPO.git /home/control/$CONFIG_REPO
```
Comment (Donovan-Medina) [inline control-services/bin/setup.sh:162]: Good catch, added repo

File: control-services/bin/setup.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (rylagek) [reply control-services/bin/setup.sh:118]: Interesting, so the way it's written now, whoever is setting up control has to add the keys to their account and their account needs proper access. That's substantially better than the setup process of adding keys to the repo owner's ssh keys. TIL. That's good

[the remark this answers is not in the document]
Comment (Donovan-Medina) [reply control-services/bin/setup.sh:118]: 👍 I learned that the hard way the other day. Still begs the question of long-term ownership over the private repo for the and implementing key management/rotation for when devs are no longer needing access to that repo or managing the control box in production.

**Review by rylagek — APPROVED** [review]

**Pull request merged**